.github/workflows/ci.yml

```diff
@@ -152,6 +152,13 @@ jobs:
           command: check
           args: --examples --features full
 
+      # TODO: prolly move it to a separate step?
```

Comment:
btw is this a typo for "probably"?

Reply:
It's not a typo: https://en.wiktionary.org/wiki/prolly

Reply:
:o